### DISCUSSION TITLE
Unable to select common Metamagics for my Magician, what am i doing wrong?

### DISCUSSION THREAD

**Discussion by Coralliafluff** [chummer5a/chummer5a#5151] [Q&A] [answered]
I am not certain what i am doing wrong, but i have recently Initiated and wanted to pick centering as my first metamagic, yet it is not available to me. I am not aware of any ruling that would make it so that i would be unable to pick centering, especially not when it is from the core rulebook. My tradition for my character is Chaos Magic and i can't really find anything that would limit my choices this much.

The only Metamagics i can pick are those from Forbidden Arcana. Could there be some rule conflict that i am unaware of?

![image](https://github.com/user-attachments/assets/82ecd5b2-ffed-4690-8537-dad72ed65b42)
![image](https://github.com/user-attachments/assets/e0e0f78a-ed37-406d-92b0-f73694cef5ec)

Is there anything that i am missing? Is there a way for me to fix this?

**Comment by Saelvarath** [comment, marked as answer]
Street Grimoire divided metamagics into High Arts (p. 143-159).

Add the Centering Art first, then you will be able to select the Metamagic.
